restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -547,10 +547,10 @@ private void invite(final Object message) throws IOException, NumberParseExcepti
                 // proxy DID or number if the outbound proxy fields are not empty in the restcomm.xml
                 if (proxyURI != null && !proxyURI.isEmpty()) {
```

Comment:
Done for now, but need to rethink about better way to add headers. Currently only duplicated code from `Call.addHeadersToMessage`